code/css/theme-default.css

```diff
@@ -0,0 +1,127 @@
+body {
+  background-color: black;
+}
+
+h1 {
+  text-align: center;
+  color: white;
+}
+
+/* box for entire calculator - needed to position the content */
+
+.rcalc-container {
+  text-align: center;
+  margin-bottom: 200px;
+}
+
+/* calculator form */
+
+.rcalc-calculator {
+  background-color: AliceBlue;
+  display: inline-block;
+  padding: 2px;
+}
+
+/* clear button */
+
+.rcalc-button-clear {
+  background-color: #681c1c;
+  border: none;
+  color: white;
+  font-size: 40px;
+  width: 60px;
+  height: 60px;
+  cursor: pointer;
+}
+
+/* result display */
+
+#id-display-main {
+  text-align: right;
+  border-style: solid;
+  border-color: #1c3968;
+  color: black;
+  background-color: white;
+  font-size: 40px;
+  width: 186px;
+  height: 55px;
+}
+
+/* error display */
+
+#id-display-info {
+  display: block;
+  text-align: center;
+  border-style: solid;
+  border-color: #1c3968;
+  border-width: 2px;
+  font-weight: bold;
+  color: red;
+  background-color: white;
+  font-size: 18px;
+  width: 250px;
+  height: 55px;
+}
+
+/* all buttons of calculator accept C */
+
+.rcalc-button {
+  border: none;
+  color: white;
+  background-color: #1c3968;
+  font-size: 40px;
+  width: 60px;
+  height: 60px;
+  cursor: pointer;
+}
+
+/* dropdown button */
+
+.rcalc-dropdown-button {
+  background-color: #1c3968;
+  color: white;
+  font-weight: bold;
+  margin-top: 25px;
+  padding: 5px;
+  font-size: 20px;
+  border: none;
+}
+
+/* the container <div> - needed to position the dropdown content */
+
+.rcalc-dropdown {
```

Comment:
Perfect. Now when you will use next dropdown and with another style ... it will be easy create another one with another style. It is not good idea use default and then rewrite it ... this rewrite all dropdowns on all places and this is undesirable. ;-)